backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -129,3 +129,11 @@ body {
   border-top: 1px solid $color-border;
   padding: 10px 0;
 }
+
+// Button Spacing
```

Comment:
Good points, thanks once again Gray.  Slotting it into the `_forms.scss` also reminded me to remove the now unused `span.or` and also let me nest the styles into the same position in the stylesheet that `span.or` had been.